### ISSUE TITLE
Color scheme broken in structure view

### ISSUE THREAD

**Issue by dhruvkb** [4lex4/intellij-platform-solarized#28]
The color scheme Solarized Dark shows black text inside the Structure panel. This is not clearly readable. The text should be shown in the default text color for the dark theme.

<img width="270" alt="Screenshot 2021-09-09 at 2 33 50 PM" src="https://user-images.githubusercontent.com/16580576/132656724-fbb95b05-1f59-4977-b58a-49391d364319.png">

**Comment by 4lex4** [comment]
Can't reproduce, the text has default text color in my ide. Did you edit the editor scheme?

**Comment by dhruvkb** [comment]
No, I haven't edited the theme colors. The 'Restore Defaults' option being greyed out confirms this.

<img width="699" alt="Screenshot 2021-09-11 at 5 12 43 PM" src="https://user-images.githubusercontent.com/16580576/132946767-3a977de6-ba91-4db5-af68-0599028a044e.png">

Can you share where this text color is defined? I can check if that is a non-default value. If this info is relevant, I'm on macOS Big Sur with the system-wide dark mode enabled.
